include/cantera/kinetics/RxnRates.h

```diff
@@ -451,6 +451,69 @@ class ChebyshevRate
     vector_fp dotProd_; //!< dot product of chebCoeffs with the reduced pressure polynomial
 };
 
+//! Langmuir-Hinshelwood reaction rate type
+class LangmuirRate
+{
+public:
+    //! return the rate coefficient type.
+    static int type() {
+        return LANGMUIR_REACTION_RATECOEFF_TYPE;
+    }
```

Comment:
I know this was written for consistency with the other classes in `RxnRates.h`, but I don't think it's actually used anywhere.